cmake/fake_tribits.cmake

```diff
@@ -104,10 +104,14 @@ FUNCTION(KOKKOS_ADD_TEST)
       NUM_MPI_PROCS 1
       ${TEST_UNPARSED_ARGUMENTS}
     )
+
+    if(TEST_TOOL)
+	    set_property(TEST ${TEST_NAME} APPEND_STRING PROPERTY ENVIRONMENT "KOKKOS_PROFILE_LIBRARY=${TEST_TOOL}")
```

Comment:
Are we assuming that `TOOL` is always a library built by Kokkos itself? Then I would do the following. Make `TOOL` a target name, not a path.
````
add_dependencies(${EXE} ${TEST_TOOL}) #make sure the exe has to build the tool
get_target_property(TOOL_LIB ${TEST_TOOL} LOCATION)
set _property(TEST ${TEST_NAME} APPEND_STRING PROPERTY ENVIRONMENT "KOKKOS_PROFILE_LIBRARY=${TEST_LIB}")
````

Reply:
Slick. I'll just implement these, they seem strict upgrades

Reply:
Turns out CMake is annoying, I implemented what I think is an equivalent answer. Let me know though

Reply:
We discussed this, to the best of my knowledge Jeremy is happy with the resolution